etcdctl/ctlv3/command/move_leader_command.go

```diff
@@ -72,6 +72,7 @@ func transferLeadershipCommandFunc(cmd *cobra.Command, args []string) {
 	}
 	if leaderCli == nil {
 		cobrautl.ExitWithError(cobrautl.ExitBadArgs, fmt.Errorf("no leader endpoint given at %v", eps))
+		return
```

Comment:
Why add this? Did the linter complain this?
It will never be executed. `cobrautl.ExitWithError` exits the process.

Reply:
OK, in that case, pls add a comment above to clarify why only adding `return` here (note that there are lots of `cobrautl.ExitWithError` usage). Not everyone read commit message.

Reply:
Interesting. I noticed this being triggered straight checking out from the main branch. I pushed without this commit, and golanci-lint works. It must be something in my local environment.